osclib/accept_command.py

```diff
@@ -56,9 +56,9 @@ def perform(self, project):
             msg = 'Accepting staging review for {}'.format(req['package'])
             print(msg)
 
-        for req in requests:
```

Comment:
I needed to merge this loop in the above one, as otherwise I do not have information about the package name.
get_package_for_request_id works based on the prj metadata, so this could potentially be an option, but sounds like an overkill (as I don't see much benefit of first accepting staging review for each package, and then the package, as opposed to doing the two accepts in one go, then moving to the next package)